io.gitlab.android_translation_layer.BaseApp.yml

```diff
@@ -0,0 +1,261 @@
+id: io.gitlab.android_translation_layer.BaseApp
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+cleanup:
+  - /include
+  - /bin/gn
+  - /bin/dx
+  - '/lib/*.a'
+  - '/lib/*.la'
+  - '/lib/libopenxr_loader.so*'
+  - '/lib/libportal-*'
+  - /lib/cmake
+  - /lib/girepository-1.0
+  - /lib/java/core-all_classes.jar
+  - /lib/java/dx.jar
+  - /lib/pkgconfig
+  - /lib/python3.11
```

Comment:
except that this runtime has Python 3.12

Reply:
I have fixed it by simply adding cleanup '*' to the python3-typogrify module